python/tvm/testing/aot.py

```diff
@@ -455,6 +465,7 @@ def _emit_main_common_includes(main_file, custom_includes):
     main_file.write("#include <math.h>\n")
     main_file.write('#include "tvm/runtime/c_runtime_api.h"\n')
     main_file.write('#include "tvm/runtime/crt/stack_allocator.h"\n')
+    main_file.write('#include "tvm/runtime/crt/module.h"\n')
```

Comment:
interesting, I thought `c_runtime_api.h` would be enough :smile_cat:

Reply:
`tvm/runtime/crt/module.h` defines `TVMModule` needed by
`TVMModule* TVMSystemLibEntryPoint(void)`.